src/components/Dialogs/Multistepper/Multistepper.style.ts

```diff
@@ -0,0 +1,81 @@
+import styled from '@emotion/styled'
+import { Text, Button } from '@/shared/components'
+import { colors, sizes, breakpoints, typography } from '@/shared/theme'
+
+type CircleProps = {
+  isFilled?: boolean
+}
+
+type isActiveProp = {
```

Comment:
```suggestion
type StyledStepInfoProps  = {
```
I know that you used it two times, but still - I think we should stick to types written in uppercase. Also, in my opinion, this is a more appropriate name here